docs/src/pages/styles/api/api.md

```diff
@@ -53,7 +53,7 @@ style rules to `makeStyles`/`withStyles` which are a function of the `Theme`.
 ```jsx
 import { makeStyles, createStyles } from '@material-ui/styles';
 
-const styles = makeStyles((theme: Theme) => createStyles({
+const useStyles = makeStyles((theme: Theme) => createStyles({
```

Comment:
This looks good. Though we manage our translations with crowdin. You don't need to change the other translations (everything that has the -somelocale suffix). Please reduce the changes to only affect api.md

Reply:
Thank you for teach me.
I understand fix point and I fixed it.
Could you re-review this PR, please.